tests/test_backend.py

```diff
@@ -2932,6 +2933,27 @@ def func(input_holder):
         for input_val in input_vals:
             self._run_test_case(func, [_OUTPUT], {_INPUT: input_val})
 
+    @skip_tf_cpu("only tf_gpu can run CudnnGPU")
+    @check_opset_min_version(11, "GRU")
```

Comment:
why opset 11?

Reply:
Tested - we could support from opset 10 with what we have already.